### DISCUSSION TITLE
小白求教 导入了 geoip 后国内网站也走vpn

### DISCUSSION THREAD

**Discussion by casillahao** [MetaCubeX/ClashMetaForAndroid#402] [Q&A]
导入了github 里找的上千星的 geoip，模式选择的“规则”（默认的不修改是啥？今天才注意到），网络设置里都是默认，所有软件都可以使用代理。在国内的小米应用商店下载软件，能看到还是走的vpn 流量显示下载速度33m/s，我理解国内流量下载时，这里应该是0 吧？有大佬帮忙答疑吗？简单说就是怎么才能实现自动切换呢，感谢![Screenshot_2024-12-05-14-53-08-420_com.xiaomi.market.jpg](https://github.com/user-attachments/assets/086e1e68-2337-412f-ba4b-7b042de1bdb2)

![Screenshot_2024-12-05-14-53-31-781_com.github.metacubex.clash.meta.jpg](https://github.com/user-attachments/assets/d6de396d-a4eb-4687-b169-a2af77b7b909)

![Screenshot_2024-12-05-14-56-39-066_com.github.metacubex.clash.meta.jpg](https://github.com/user-attachments/assets/4591a13d-70ac-4b72-93b2-a071bbfa01ff)

**Comment by dreamoeu** [comment]
使用ClashForAndroid,它这个直连下载流量是不会显示下载速度

**Reply by xiaofeng0223** [reply]
麻烦问下你们这些节点是怎么来的，在哪买的呀

**Reply by PiliPlan** [reply]
各大机场呗，不过你能在这里回复，不就说明了你也有节点么……

**Reply by Oliver649453661** [reply]
> 各大机场呗，不过你能在这里回复，不就说明了你也有节点么……

大哥，有没有值得推荐的机场，包总量或者不算太贵的包年，平时就用用gpt和github

**Reply by PiliPlan** [reply]
我用的10块一个月的，情况好的时候可以看1080视频和720直播，差的时候降一档，我也不知道这水平算不算好，不过用了几年没跑路，也就一直用了，你要的话留个邮箱我发你。

**Reply by Oliver649453661** [reply]
> 我用的10块一个月的，情况好的时候可以看1080视频和720直播，差的时候降一档，我也不知道这水平算不算好，不过用了几年没跑路，也就一直用了，你要的话留个邮箱我发你。

好的，谢谢大哥了y649453661@gmail

**Reply by PiliPlan** [reply]
发了，没看到就在垃圾箱。

**Comment by qingchenyu925** [comment]
关闭clash，然后把允许所有应用访问改成自定义已选择的，把需要用的应用勾上就行
